src/views/HomePage.vue

```diff
@@ -17,6 +17,26 @@
             @select-course="onSelectCourse">
         </course-list>
         <course-create @create-course="onCreateCourse"></course-create>
+        <button class="bookmarklet-button"
+          @click="openBookmarkletModal"
+        >
+        Enable NB Sidebar
```

Comment:
suggestion: how about labeling this as "NB Bookmarklet Instructions" or something similar.